### PR TITLE
fix video get learning_rate error

### LINKED ISSUE #2169: video AttentionCluster训练报错
在paddle1.4.1上，video AttentionCluster训练报错，报错如下：
Traceback (most recent call last):
  File "train.py", line 251, in <module>
    train(args)
  File "train.py", line 241, in train
    test_metrics=valid_metrics)
  File "/ssd1/xiege/model_4.29/models-develop/PaddleCV/video/tools/train_utils.py", line 104, in train_with_pyreader
    lr = fluid.global_scope().find_var("learning_rate").get_tensor()
AttributeError: 'NoneType' object has no attribute 'get_tensor'

### PR DESCRIPTION
**Pull request by heavengate**
fix https://github.com/PaddlePaddle/models/issues/2169

### CODE REVIEW

File: PaddleCV/video/tools/train_utils.py [hunk 1]
```diff
@@ -9,6 +9,26 @@
 logger = logging.getLogger(__name__)
 
 
+def log_lr_and_step():
+    try:
+        # In optimizer like Adam, lr_var name is 'learning_rate_0',
+        # in decay lr from learning_rate_scheduler, lr_var name is
+        # 'learning_rate', better impliment is required here
```
Comment (SunGaofeng) [inline PaddleCV/video/tools/train_utils.py]: description 改下吧，不是因为Adam里面lr_var name 是'learning_rate_0'，是在没有设置decay的时候，lr_var name是 'learning_rate_0'，没有LR_DECAY_COUNTER

File: PaddleCV/video/tools/train_utils.py [hunk 2]
```diff
@@ -9,6 +9,28 @@
 logger = logging.getLogger(__name__)
 
 
+def log_lr_and_step():
+    try:
+        # In optimizers, if learning_rate is set as constant, lr_var 
+        # name is 'learning_rate_0', if learning_rate is set 
+        # as decayed values from learning_rate_scheduler, lr_var 
+        # name is 'learning_rate', and not lr_count record, 
+        # better impliment is required here
```
Comment (SunGaofeng) [inline PaddleCV/video/tools/train_utils.py:19]: # In optimizers, if learning_rate is set as constant, lr_var
	        # name is 'learning_rate_0', and iteration counter is not recorded. If learning_rate is set 
	        # as decayed values from learning_rate_scheduler, lr_var 
	        # name is 'learning_rate', and iteration counter is recorded with name '@LR_DECAY_COUNTER@', 
	        # better impliment is required here

**Review by SunGaofeng — APPROVED** [review]

File: PaddleCV/video/tools/train_utils.py (hunk 1 above)
Comment (heavengate) [reply PaddleCV/video/tools/train_utils.py]: Done

File: PaddleCV/video/tools/train_utils.py (hunk 2 above)
Comment (heavengate) [reply PaddleCV/video/tools/train_utils.py:19]: Done. Thanks

**Pull request merged**